tests/test_server.py

```diff
@@ -11,169 +11,194 @@
 - "new_server": starting a new server with any configurations on each test.
 
 """
-from __future__ import print_function
-
-from collections import namedtuple
 import contextlib
-import functools
+import itertools
 import os
-import subprocess
+import shutil
+import socket
 import sys
-import tempfile
 import time
+from collections import namedtuple
 from pathlib import Path
 from shlex import split
 from subprocess import Popen
-from textwrap import dedent
+from urllib.error import URLError
+from urllib.request import urlopen
 
-try:
-    from urllib.request import urlopen
-except ImportError:
-    from urllib import urlopen
-
-from py import path  # @UnresolvedImport
 import pytest
 
-
 # ######################################################################
 # Fixtures & Helper Functions
 # ######################################################################
 
 
-_BUFF_SIZE = 2 ** 16
-_port = 8090
-SLEEP_AFTER_SRV = 3  # sec
+CURRENT_PATH = Path(__file__).parent
+ports = itertools.count(10000)
```

Comment:
Could we make this something like `range(8000, 10000)`, so that it won't try to bind privileged ports?

Reply:
So currently it starts at 10000, so it shouldn't run into any privileged ports, although a range object also works I guess :)

Reply:
oh I see! that's my mistake then. I have never used `count`, and I assumed it worked like range